pallets/hermes-governance-platform/src/benchmarking.rs

```diff
@@ -0,0 +1,246 @@
+//! Hermes governance platform module benchmarking.
```

Comment:
Not sure, if it is important during this merge, but there is missing copyright in hermes-governance-platform pallet files

Reply:
As I remember, hermes is not our code and I'm not sure we can add our copyright header for these files. The same logic for ceres.